doi/src/main/java/ca/nrc/cadc/doi/DoiAction.java

```diff
@@ -93,9 +117,11 @@ public abstract class DoiAction extends RestAction {
     public static final URI DOI_VOS_REQUESTER_PROP = URI.create("ivo://cadc.nrc.ca/vospace/doi#requester");
     public static final URI DOI_VOS_STATUS_PROP = URI.create("ivo://cadc.nrc.ca/vospace/doi#status");
     public static final URI DOI_VOS_JOURNAL_PROP = URI.create("ivo://cadc.nrc.ca/vospace/doi#journalref");
+    public static final URI DOI_VOS_TITLE_PROP = URI.create("ivo://cadc.nrc.ca/vospace/doi#title");
 
     public static final String STATUS_ACTION = "status";
     public static final String MINT_ACTION = "mint";
+    public static final String SEARCH_ACTION = "search";
     public static final String JOURNALREF_PARAM = "journalref";
     public static final String DOI_GROUP_PREFIX = "DOI-";
     public static final String TEST_DOI_GROUP_PREFIX = "TEST.DOI-";
```

Comment:
Missed this. I think this was originally defined to use a common name for test groups, making them easier to find and delete afterwards. But cleaning up afterwards isn't something we tend to do, and defining test variables in a class isn't the right way to solve that problem. The Group prefix is configured in the doi.properties file. For local testing it can be set to the developers choice, and the int-test README could be updated to suggest a common group prefix used for int-tests.